docs/x509/verification.rst

```diff
@@ -276,6 +276,32 @@ the root of trust:
 
         :returns: A new instance of :class:`PolicyBuilder`
 
+    .. method:: ca_extension_policy(new_policy)
+
+        .. versionadded:: 44.0.0
+
+        Sets the CA extension policy for the verifier.
+        If this method is not called, the default CA extension policy that 
+        follows the CA/B Forum guidelines is used.
+
+        :param ExtensionPolicy new_policy: The CA extension policy to use. 
+            Use :class:`ExtensionPolicyBuilder` to create the policy.
+
+        :returns: A new instance of :class:`PolicyBuilder`
+
+    .. method:: ee_extension_policy(new_policy)
```

Comment:
Does it actually make sense for these to be two separate methods? What's the cirucmstance where you want to set one but not the other?

Reply:
Well, my use case is to relax some of the extension policy requirements for EE certificates, namely - allow different extended key usages and to allow certificates without SANs since we don't always have them. All our CA certificates however comply to the default extension policy, so there's no need to adjust that for me. 

A single `set_extension_policies(ee_ext_policy, ca_ext_policy)` method would work, but would make things a bit more verbose when you only want to set one of the two: `set_extension_policies(custom_policy, ExtensionPolicyBuilder.webpki_defaults_ca().build()`

Reply:
If we do consider the single-method option I would also consider changing 
```python
ExtensionPolicyBuilder.webpki_defaults_ca().require_not_present(SubjectAlternativeName.oid).build()
```
to
```python
ExtensionPolicy.webpki_defaults_ca().require_not_present(SubjectAlternativeName.oid)
```
Implementation-wise the build method is only there to make the API make sense from an outside perspective, as internally the builder itself is just a policy wrapper with some methods. However I understand that this might theoretically make the api a bit more confusing as it still uses the builder pattern without explicitly signalling that in the type name.

Reply:
Not needing an explicit `build()` is ok with me, regardless of what decision we make about seperate EE/CA policies.

Reply:
Without requiring a `build()` call (and with the name shortened to `ExtensionPolicy`, I think even I prefer the single method version. `ExtensionPolicy. webpki_defaults_ca()` is much shorter, and it forces the user code to be more explicit whenever any changes to extension policies are made, which is a good thing I think.

Reply:
Yes, I'm persuaded

Reply:
Just a note, I guess it'll have to be `PolicyBuilder().extension_policies(...)` instead of `PolicyBuilder().set_extension_policies(...)` since that's the existing convention.

Reply:
Sure